django_auth_adfs/config.py

```diff
@@ -69,6 +77,7 @@ def __init__(self):
         self.TIMEOUT = 5
         self.USERNAME_CLAIM = "winaccountname"
         self.JWT_LEEWAY = 0
+        self.FAILED_RESPONSE_FUNCTION = default_failed_response
```

Comment:
Could we split the `default_failed_response` out in a file? For instance`default_error_view.py`. Alternatively do as @sondrelg suggested to me and make it a lambda? 

```suggestion
        self.FAILED_RESPONSE_FUNCTION = lambda request, error_message, status: render(
            request, 'django_auth_adfs/login_failed.html', {'error_message': error_message}, status=status
        )
```

Reply:
Switched to lambda.